nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/nxql/TestNxqlConversion.java

```diff
@@ -19,7 +19,9 @@
 
 package org.nuxeo.elasticsearch.test.nxql;
 
-import org.apache.commons.lang.SystemUtils;
+import java.util.concurrent.TimeUnit;
+import javax.inject.Inject;
```

Comment:
You shouldn't need to change import order, see https://github.com/nuxeo/nuxeo/blob/master/tools/nuxeo.importorder